### PR TITLE
Dashboard: Add checks for logos dimensions

### PR DESCRIPTION
**Pull request by dmmulroy**
## Summary

Adds error messages in the case that uploaded logos are smaller than 96x96.

## User-facing changes

- If you upload one image, and it is smaller than 96x96, the following error message will display: `Sorry, this file's dimensions are too small. Make sure your logo is larger than 96x96.`
- If you upload two or more images, and one or more are smaller than 96x96, the following error message will display: `Sorry, one or more of these files dimension's are too small. Make sure your logos are all larger than 96x96.`

## Testing Instructions

- Upload a single image that is smaller than 96x96 and verify that you see the first error message above.
- Upload two or more images, where at least one is smaller than 96x96, and verify that you see the second 

#4147

### CODE REVIEW

**Comment by carloskelly13** [comment]
Works well!

Screen Shot 2020-09-02 at 1 08 38 PM
Screen Shot 2020-09-02 at 1 08 54 PM

**Review by carloskelly13 — APPROVED** [review]
This is tricky to test but I think we can create a Mock Image object with a blob of random data and some metrics.

File: assets/src/dashboard/app/views/editorSettings/index.js [hunk 1]
```diff
@@ -146,6 +164,54 @@ function EditorSettings() {
...
+              )
+            : sprintf(
+                /* translators: 1 = minimum width, 2 = minimum height */
+                __(
+                  "Sorry, one or more of these files dimension's are too small. Make sure your logos are all larger than %1$dx%2$d.",
+                  'web-stories'
+                ),
+                MIN_IMG_WIDTH,
+                MIN_IMG_HEIGHT
+              );
```
Comment (swissspidy) [inline assets/src/dashboard/app/views/editorSettings/index.js]:
```suggestion
            : sprintf(
                /* translators: %s: image dimensions in pixels. */
                __(
                  'Sorry, one or more files are too small. Make sure your logos are all larger than %s.',
                  'web-stories'
                ),
                sprintf('%1$dx%2$dpx', MIN_IMG_WIDTH, MIN_IMG_HEIGHT)
              );
```

File: assets/src/dashboard/app/views/editorSettings/index.js [hunk 2]
```diff
@@ -146,6 +164,54 @@ function EditorSettings() {
...
+          files.length === 1
+            ? sprintf(
+                /* translators: 1 = minimum width, 2 = minimum height */
+                __(
+                  "Sorry, this file's dimensions are too small. Make sure your logo is larger than %1$dx%2$d.",
+                  'web-stories'
+                ),
+                MIN_IMG_WIDTH,
+                MIN_IMG_HEIGHT
+              )
```
Comment (swissspidy) [inline assets/src/dashboard/app/views/editorSettings/index.js]:
```suggestion
            ? sprintf(
                /* translators: 1 = minimum width, 2 = minimum height */
                __(
                  'Sorry, this file is too small. Make sure your logo is larger than %s.",
                  'web-stories'
                ),
                sprintf('%1$dx%2$dpx', MIN_IMG_WIDTH, MIN_IMG_HEIGHT)
              )
```

File: assets/src/dashboard/app/views/editorSettings/index.js [hunk 3]
```diff
@@ -37,6 +37,8 @@ import { Main, Wrapper } from './components';
 import PublisherLogoSettings from './publisherLogo';
 
 const ACTIVE_DIALOG_REMOVE_LOGO = 'REMOVE_LOGO';
+const MIN_IMG_HEIGHT = 96;
+const MIN_IMG_WIDTH = 96;
```
Comment (swissspidy) [inline assets/src/dashboard/app/views/editorSettings/index.js]: Maybe rename so it's clear that these are for publisher logos?

Could be a candidate for `assets/src/dashboard/constants/index.js` too as these might be useful elsewhere in the future.

**Review by mariano-formidable — APPROVED** [review]
I agree with Pascal's error message changes, but functionally and code-ically it works!  Nice job!!

**Review by BrittanyIRL — APPROVED** [review]
LGTM!

**Pull request merged**